front_end/index.html

```diff
@@ -30,7 +30,10 @@ <h3>Create a new todo</h3>
             Completion date
             <input name="completionDate" type="date" required />
           </label>
-          <button title="Create a new todo">➕ Create!</button>
+          <label>
```

Comment:
What does adding this label solve? A button shouldn't need a label since the button content is the label and is contained within the button already!